backend/source/source/migrations/0020_traktsource.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.2 on 2022-05-07 14:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('source', '0019_delete_rsyncdestination'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TraktSource',
+            fields=[
+                ('key', models.SlugField(max_length=80, primary_key=True, serialize=False)),
+                ('consumer_key', models.CharField(max_length=100)),
+                ('consumer_secret', models.CharField(max_length=100)),
+                ('access_token', models.CharField(blank=True, max_length=100)),
+                ('refresh_token', models.CharField(blank=True, max_length=100)),
+                ('access_token_created', models.IntegerField(null=True)),
```

Comment:
Integer? Shouldn't this be a date?

Reply:
From what I can see it uses milliseconds (since 1970) for date/time related fields in OAuth (e.g. see [Microsoft](https://docs.microsoft.com/en-us/machine-learning-server/operationalize/how-to-manage-access-tokens#create-tokens). I've not yet come across a different field type for this

Reply:
Django has the more appropriate DateTimeField, which I use for other dates. It will get serialised properly by the API, as a bonus.